Resources/Prototypes/Entities/Mobs/Species/slime.yml

```diff
@@ -96,6 +96,34 @@
     - type: HumanoidAppearance
       hairMatchesSkin: true
       hairAlpha: 0.5
+    - type: Damageable
+      damageContainer: Biological
+      damageModifierSet: Slime
+    - type: Barotrauma
+      damage:
+        types:
+          Blunt: 1.4 #per second, scales with pressure and other constants. Twice as much as humans.
+    - type: Reactive
+      groups:
+        Flammable: [ Touch ]
+        Extinguish: [ Touch ]
+      reactions:
+      - reagents: [ Water, SpaceCleaner ]
+        methods: [ Touch ]
+        effects:
+        - !type:WashCreamPieReaction
+      - reagents: [ Water ]
+        methods: [ Touch ]
+        effects:
+        - !type:HealthChange
+          scaled: true
+          damage:
+            types:
+              Heat: 2
```

Comment:
Heat seems silly, but I guess it is currently the best option to represent something like acid "melting away" your body until damage types are rejigged.

Reply:
Are there other ways of applying water to people? Because the spray bottle damage output seems fine, but just concerned someone could throw a bucket at someone to kill them or something. AFAICT walking through puddles does nothing.

Reply:
fire extinguishers apply water which is what i balanced around, and it works fine